packages/atlas/src/views/viewer/VideoView/VideoView.styles.ts

```diff
@@ -62,6 +62,8 @@ const getPlayerContainerCinematicStyles = ({ cinematicView }: CinematicView) =>
         }
       `
     : css`
+        padding-top: 56.3%;
```

Comment:
For consistency with other usages:

```suggestion
        padding-top: 56.25%;
```